waspc/packages/deploy/src/providers/shared/helpers.ts

```diff
@@ -0,0 +1,108 @@
+import fs from 'fs';
+import path from 'node:path';
+import { $ } from 'zx';
+import { Command } from 'commander';
+import { cd, ProcessOutput, Shell } from 'zx';
+import { exit } from 'process';
+
+// For some reason, the colors from the chalk package wouldn't
+// show up when run as a subprocess by the Wasp CLI. This works.
+export function waspSays(str: string): void {
```

Comment:
We should implement `waspWarns` and `waspScreams` for warnings and errors as we do in Haskell :) 

Probably out of scope for this PR but just a thought I had.